src/DataSource/NextrasDataSource.php

```diff
@@ -41,6 +41,11 @@ class NextrasDataSource extends FilterableDataSource implements IDataSource
 	 */
 	private $dbalCollectionClass;
 
+	/**
+	 * @var int
+	 */
+	private $ormVersion;
```

Comment:
@martenb Thank you for your contribution but I do not think it is a good idea to indroduce new property ormVersion next to dbalCollectionClass. I think you can rewrite the code and only use one of them because they serve the same purpose.

Reply:
@vody105 ok, changed